media-server/media_server.sh

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#
+# Author: Fabian Solano <fabian.solano@ridgerun.com>
+# 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+
+# Load settings
+source config.txt
+
+# Create pipelines
+echo "Creating camera pipeline"
```

Comment:
Can we drop Gstreamer specific concepts? Let's print message more generic. Something like: `Initializing camera capture`